WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -0,0 +1,17 @@
+<% competitions.each_with_index do |competition, index| %>
+  <tr class="<%= competition.is_over? ? "past" : "not-past" %>">
+    <td><%= competition.year %></td>
+    <td><%= date_range(competition.start_date, competition.end_date, separator: '-', show_year: false) %></td>
+    <td>
+      <%= link_to competition.name, competition_path(competition) %>
+    </td>
+    <td>
+      <% if Result.exists?(competitionId: competition.id) %>
```

Comment:
This should go in `Competition.results_uploaded?`, right? I thought you weren't planning to keep this because it's still too slow, though?